utils.js

```diff
@@ -65,6 +64,18 @@ export async function initializeDiscordClient() {
         console.log("discord client is ready");
         DISCORD_CLIENT_READY = true;
     });
+
+    // eslint-disable-next-line no-promise-executor-return
+    const snooze = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
+    let waitInterval = 100;
+
+    while (!DISCORD_CLIENT_READY) {
+        await snooze(100);
+        if (waitInterval % 1000 === 0) console.log("waiting for the client to be ready");
```

Comment:
After https://github.com/kajahno/elprimobot/pull/55 you won't need to do this.

You'll only have to do `getDiscordClient()` whenever you need a new instance of discord that it's ready.

Reply:
Yeah we can refactor once that's done